chillTime.py

```diff
@@ -46,22 +48,14 @@ def __init__(self, algorithm='simple'):
         # Save the algorithm to be used.
         self.algorithm = algorithm.lower()
 
-    sql_stmts = {
-###*** Need to make sure the sqlLite statement is correct, since I don't use it
-        'sqlite': "SELECT outTemp FROM {table} WHERE dateTime BETWEEN {start} AND {stop}",
-        'mysql': "SELECT outTemp FROM {table} WHERE dateTime BETWEEN {start} AND {stop}"
-    }
-
     def get_scalar(self, obs_type, record, db_manager):
-        # Calculate 'chillTime'. For everything else, raise an exception UnknownType
+        """Calculate 'chillTime'. For everything else, raise an exception UnknownType"""
+
         if obs_type != 'chillTime':
             raise weewx.UnknownType(obs_type)
 
         # We need outTemp and interval in order to do the calculation.
-        if 'outTemp' not in record or record['outTemp'] is None:
-            raise weewx.CannotCalculate(obs_type)
-        if 'interval' not in record or record['interval'] is None:
-            raise weewx.CannotCalculate(obs_type)
+        if record.get('outTemp') is None or record.get('interval') is None:
 
```

Comment:
Should there be `raise weewx.CannotCalculate(obs_type)` here?

Reply:
Yes, there should be. I warned you that it wasn't tested! :-)